src/daemon/CMakeLists.txt

```diff
@@ -56,6 +56,22 @@ elseif (LINUX)
     find_package(UDEV REQUIRED)
 endif ()
 
+# Declare options with defaults to allow overrides in debian packaging
+set(UDEV_RULES_PATH
+    "/etc/udev/rules.d"
```

Comment:
Rules are now being installed by default in `/usr/lib/udev/rules.d/` so you may not need this override anymore.